src/Engine/Component/RigidBody.cpp

```diff
@@ -41,23 +40,37 @@ namespace Component {
     }
 
     glm::vec3 RigidBody::Position() const {
-        btTransform trans;
-        rigidBody->getMotionState()->getWorldTransform(trans);
+        btTransform trans = rigidBody->getWorldTransform();
         return Physics::btToGlm(trans.getOrigin());
     }
 
     void RigidBody::Position(const glm::vec3& pos) {
-        btTransform trans;
-        rigidBody->getMotionState()->getWorldTransform(trans);
+        btTransform trans = rigidBody->getWorldTransform();
         trans.setOrigin(Physics::glmToBt(pos));
         rigidBody->setWorldTransform(trans);
     }
 
+    glm::quat RigidBody::Orientation() const {
```

Comment:
I think GetOrientation and SetOrientation would be clearer than two methods with that same name that do different things.